admin_pages/transactions/Transactions_Admin_Page.core.php

```diff
@@ -1105,8 +1082,8 @@ public function txn_details_meta_box()
         }
 
         $this->_template_args['txn_details']['registration_session']['value'] = $this->_transaction
-            ->get_first_related('Registration')
-            ->get('REG_session');
+            ->primary_registration()
```

Comment:
I have the same observation here as made earlier, `primary_registration` does more logic than `get_first_related( 'Registration' )` does. That may be intentional, but I just want to make sure.  I get that this likely will be an improvement because `primary_registration()` _ensures_ we're only getting the primary registration (no matter what its order was in saving).

Reply:
Well both the primary and additional registrations can be changed, and a transaction can have registrations from multiple sessions, so it could be debated what session ID to display for the transaction, but it will still likely always make the most sense to use the session ID for the primary registrant. ie: why would you use the session ID from any other registrant?

Reply:
> ie: why would you use the session ID from any other registrant?

I agree, but we're not introducing new code, we're changing old code (as flawed as it is).  So along with considering the _improvement_ this brings, I think it is also important to consider what effects this might have on code that was written against this flawed code (and might have made its own adjustments).  Again, just a sanity check.

Reply:
there's no side effects... this is for a template arg, and there's a infinitesimally small chance at best that someone could be using this in some other way that would be affected.

Here is where the session ID is being used:
![transaction session details](https://user-images.githubusercontent.com/1751030/51207239-1ca66d80-18bf-11e9-9246-536523947080.png)

which is normally hidden from view and probably only ever looked at by support staff